sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_v1/query.sql

```diff
@@ -29,6 +29,7 @@ WITH todays_metrics AS (
     search_with_ads,
     uri_count,
     active_hours_sum,
+    windows_build_number
```

Comment:
We wouldn't need this field. but the UDF applied to the OS Version only when the OS = Windows, right?